spec/var-op-len.test.sh

```diff
@@ -113,21 +113,21 @@ true
 -1
 -1
 -1
-[ stdin ]:3: warning: Invalid start of UTF-8 character
-[ stdin ]:3: warning: Invalid start of UTF-8 character
-[ stdin ]:3: warning: Invalid start of UTF-8 character
-[ stdin ]:3: warning: Invalid UTF-8 continuation byte
-[ stdin ]:3: warning: Invalid start of UTF-8 character
-[ stdin ]:3: warning: Invalid start of UTF-8 character
-[ stdin ]:3: warning: Invalid UTF-8 continuation byte
-[ stdin ]:3: warning: Invalid UTF-8 continuation byte
-[ stdin ]:3: warning: Invalid start of UTF-8 character
-[ stdin ]:3: warning: Invalid start of UTF-8 character
-[ stdin ]:3: warning: Invalid UTF-8 continuation byte
-[ stdin ]:3: warning: Invalid UTF-8 continuation byte
-[ stdin ]:3: warning: Invalid UTF-8 continuation byte
-[ stdin ]:3: warning: Invalid start of UTF-8 character
-[ stdin ]:3: warning: Invalid start of UTF-8 character
+[ stdin ]:3: warning: Utf8 Error: Bad Encoding at 0
```

Comment:
Some cosmetic issues

- Utf8 -> UTF-8   -- should be able to `sed` this out in most places
- "at 5" -> how about "at offset 5 in string" ?

Or I'm not sure if this helps, but "at offset 5 in string of length 10" might narrow down the possibilities for readers


----

I wanted to  integrate these errors with the `loc_t`  system, but that's separate from this change